src/components/SignInSteps/ExtensionStep.tsx

```diff
@@ -21,6 +14,10 @@ const ExtensionStep: React.FC<ExtensionStepProps> = ({ nextStepPath }) => {
   const navigate = useNavigate()
   const step = useRouterQuery('step')
   const { extensionConnected } = useUser()
+  const [openEnableExtensionDialog, closeEnableExtensionDialog] = useDialog({
+    additionalActionsNode: <PolkadotExtensionRejected />,
```

Comment:
This is a bit of a hack to use a custom body right? Maybe we should change this line https://github.com/Joystream/atlas/blob/ff0f6d1cfbc04ab338e1e112697ef5fbabaf0e7e/src/components/Dialogs/MessageDialog/MessageDialog.tsx#L11

and allow `description` to be `React.ReactNode` instead? Then we can pass the component as `description` (ideally that'd be `body` instead but not change for now)

Reply:
> This is a bit of a hack to use a custom body right? Maybe we should change this line
> 
> https://github.com/Joystream/atlas/blob/ff0f6d1cfbc04ab338e1e112697ef5fbabaf0e7e/src/components/Dialogs/MessageDialog/MessageDialog.tsx#L11
> 
> and allow `description` to be `React.ReactNode` instead? Then we can pass the component as `description` (ideally that'd be `body` instead but not change for now)

Yes, that's kind of a hack. 
Hmm I see, we could go for a description instead for now.